SPLF Auxiliary ghostrole

hazelrat opened the pull request:
Ghostrole in the works, putting up as WIP so any thoughts can be put forward. Further details will be written up once up for review.

hazelrat commented:
!wip

hazelrat commented:
!review

DreamySkrell commented:
![image](https://github.com/user-attachments/assets/38c53e4c-d022-4b42-b315-37db151776c3)
why permament light? why not just a normal light
or even no light as it'll get lit by the engine room lights

DreamySkrell commented:
![image](https://github.com/user-attachments/assets/574d37e3-1576-4a41-98d4-58d9ebe4ca21)
should be lattice under these pipes so they don't look like they're just floating not attached to anything

DreamySkrell commented:
![image](https://github.com/user-attachments/assets/39c39c5f-68f5-4601-85eb-6bee40ca63fa)
suggestion (hehe)
add a button to this door so it can be locked from the outside
so it could be used as a prisoner or hostage cell

hazelrat commented:
Implemented suggestions up to now, the external starlight perma light sources are sometimes needed in big extended areas of dynamic space for the lighting to not become pure darkness, but I think it should be essentially fine here.

Also added a glass airlock to the freezer so you can talk to your very frigid prisoners through it. Poor hostages.

Was tempted to add flagpatches for various Solarian nations, but really the crew of this ship should be 90% from the CRZ, either Hang Tuah's Rest (generic Sol accent), Mictlan, or another planet.
